README.md

```diff
@@ -1,76 +1 @@
-[![Build Status](https://img.shields.io/github/actions/workflow/status/tng/jgiven/jgiven_branch_and_pr_build.yml?branch=master)](https://github.com/TNG/JGiven/actions/workflows/jgiven_branch_and_pr_build.yml)
```

Comment:
are those fancy badges out?
I kinda like them

Reply:
I had essentially removed the entire `README` on the `gh-pages` branch as long as it was out of sync with the `master` branch, to avoid redunant (and outdate) information. If you prefer to keep those branches synchronized, that commit should be dropped, of course.

Reply:
I've dropped the commit touching `README.md`.